src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/Order.sql

```diff
@@ -15,6 +15,7 @@
     LastUpdatedByName NVARCHAR(256) NULL,     
 
     CONSTRAINT FK_Order_OrderStatusId_OrderStatus_OrderStatusId FOREIGN KEY (OrderStatusId) REFERENCES OrderStatus (OrderStatusId), 
-    CONSTRAINT FK_Order_AddressId_Address_AddressId FOREIGN KEY (OrganisationAddressId) REFERENCES Address (AddressId),
+    CONSTRAINT FK_Order_OrganisationAddressId_Address_AddressId FOREIGN KEY (OrganisationAddressId) REFERENCES Address (AddressId),
+    CONSTRAINT FK_Order_OrganisationBillingAddressId_Address_AddressId FOREIGN KEY (OrganisationBillingAddressId) REFERENCES Address (AddressId),
     CONSTRAINT FK_Order_OrganisationContactId_Contact_ContactId FOREIGN KEY (OrganisationContactId) REFERENCES Contact (ContactId)

```

Comment:
I think the FK names are a tad long and slightly confusing although I'm struggling to offer any alternatives. My preference would be to try to simplify them somehow. Perhaps something like below?

```sql
FK_Order_OrderStatus
FK_Order_OrganisationAddress
FK_Order_OrganisationBillingAddress
FK_Order_OrganisationContact
```

Reply:
👍